src/server/rpc/procedures/ConnectN/ConnectN.js

```diff
@@ -0,0 +1,365 @@
+// This is the ConnectN RPC. It will maintain the game board and can be queried
+// for win/tie/ongoing as well as turn
+
+'use strict';
+
+var R = require('ramda'),
+    debug = require('debug'),
+    log = debug('NetsBlox:RPCManager:ConnectN:log'),
+    trace = debug('NetsBlox:RPCManager:ConnectN:trace'),
+    Constants = require('../../../../common/Constants'),
+    info = debug('NetsBlox:RPCManager:ConnectN:info');
+
+/**
+ * ConnectNRPC - This constructor is called on the first request to an RPC
+ * from a given room
+ *
+ * @constructor
+ * @return {undefined}
+ */
+var ConnectNRPC = function() {
+    this.board = ConnectNRPC.getNewBoard();
+    this._winner = null;
+    this.lastMove = null;
+};
+
+/**
+ * Return the path to the given RPC
+ *
+ * @return {String}
+ */
+ConnectNRPC.getPath = function() {
+    return '/connectn';
+};
+
+/**
+ * This function is used to expose the public API for RPC calls
+ *
+ * @return {Array<String>}
+ */
+ConnectNRPC.getActions = function() {
```

Comment:
How does the user know if he/she won? it doesn't appear to have a method for detecting the winner